stations.csv

```diff
@@ -5862,7 +5862,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 7473;Bonn Hbf;bonn-hbf;8015485;;50.732007;7.097136;7545;DE;Europe/Berlin;f;t;f;t;f;t;DEBEA;;f;;f;8000044;t;;f;;f;;f;;f;;f;8000044;t;;f;;f;;;;f;108000044;t;;f;;f;DEBOH;f;;f;f;;;;;;;;;;;;ボン;본;;Bona;Бонн;;;波恩;urn:trainline:public:nloc:csv7473;
 7474;Hamburg Hbf;hamburg-hbf;8001071;;53.552732;10.006908;7626;DE;Europe/Berlin;f;t;f;t;f;t;DEBEB;;f;;f;8002549;t;;f;;f;;f;;f;;f;8002549;t;;f;;f;;;;f;108001071;t;;f;;f;DEHGH;f;;f;f;;;;Hamburgo;Hambourg;Amburgo;;;Hamburk;Hamborg;;ハンブルク;함부르크;;Hamburgo;Гамбург;;;汉堡市;urn:trainline:public:nloc:csv7474;
 7475;Düsseldorf Hbf;dusseldorf-hbf;8008094;;51.21996;6.794316;7578;DE;Europe/Berlin;f;t;f;t;f;t;DEBEC;;t;;f;8000085;t;;f;;f;;f;;f;;f;8000085;t;;f;;f;;;;f;108008094;t;8040260;f;;f;DEDUH;t;;f;f;;;;;;;;;;;;デュッセルドルフ;뒤셀도르프;;;Дюссельдорф;;;杜塞尔多夫;urn:trainline:public:nloc:csv7475;
-7476;Lindau reutin;lindau-reutin;8000467;;;;29006;DE;Europe/Berlin;f;f;f;f;f;f;DEBED;;f;;f;8003693;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;リンダウ;;;;Линдау;;;林道;urn:trainline:public:nloc:csv7476;
+7476;Lindau reutin;lindau-reutin;8000467;;47.5524552;9.7031148;29006;DE;Europe/Berlin;f;f;f;t;f;f;DEBED;;f;;f;8003693;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;リンダウ;;;;Линдау;;;林道;urn:trainline:public:nloc:csv7476;
```

Comment:
Could you update the formatting of the name to ```Lindau-Reutin```?

Reply:
Checking what DB says on their website